hms-api/src/rest/HackathonPreviewResponse.ts

```diff
@@ -1,34 +1,37 @@
 /* eslint-disable require-jsdoc */
 
-import Uuid from '../util/Uuid';
-import Hackathon from '../repository/domain/Hackathon';
+import Uuid from "../util/Uuid";
+import Hackathon from "../repository/domain/Hackathon";
 
 class HackathonPreviewResponse {
   id: Uuid;
   title: string;
+  description: string;
   startDate: Date;
   endDate: Date;
 
   constructor(
-      id: Uuid,
-      title: string,
-      startDate: Date,
-      endDate: Date,
+    id: Uuid,
+    title: string,
+    description: string,
+    startDate: Date,
+    endDate: Date
   ) {
     this.id = id;
     this.title = title;
+    this.description = description;
     this.startDate = startDate;
     this.endDate = endDate;
   }
 
-  static from = (
-      hackathon: Hackathon,
-  ): HackathonPreviewResponse => new HackathonPreviewResponse(
```

Comment:
the old form of having it on 3 lines is the one the backend uses

Reply:
so the backend differs depending how you write it? :O